lib/backend/k8s/resources/profile.go

```diff
@@ -128,6 +128,9 @@ func (c *profileClient) Get(ctx context.Context, key model.Key, revision string)
 	rk := key.(model.ResourceKey)
 	if rk.Name == "" {
 		return nil, fmt.Errorf("Profile key missing name: %+v", rk)
+	} else if rk.Name == "allow" {
```

Comment:
I think to do this properly we also need to return the profile in List() and Watch().

Reply:
Thanks! List() calls Get() so that looks good but Watch() I've added a conditional to return an error

Reply:
I don't think that's right - List() only calls Get() when an exact Name has been passed in.

For "list all" calls, it does not call Get().